webapp/client/cypress/integration/personAHasDisability.spec.js

```diff
@@ -70,9 +75,9 @@ describe("PersonAHasDisability", () => {
       cy.url().should("include", "/lotse/step/person_a");
     });
 
-    it("Should link forward to person b page", () => {
+    it("Should link forward to person a pauschbetrag page", () => {
       cy.get("button[type=submit]").click();
-      cy.url().should("include", "/lotse/step/person_b");
+      cy.url().should("include", "/lotse/step/person_a_requests_pauschbetrag");
```

Comment:
I think I do not really get it.
 In line 62 it says "for joint taxes". Before the pauschbetrag page, there was different behavior for joint and single taxes but now I do not think there is. I think we can just get rid of this differentiation here and have that in a new functional test for the pauschbetrag page itself?
Furthermore, I do not really understand why it always links to the person_a_requests_pauschbetrag. Is that because the person_a_requests_pauschbetrag will be skipped if we say no on this page?
If so, shouldn't we click no before submitting and require that input?

Reply:
I think person_a_requests_pauschbetrag links with person_b / telephone_number, but not necessary here because we testing personAHasDisability. If person_a_has_disability = no then the person_a_requests_pauschbetrag will be skipped. 

The context is given:
cy.request("POST", "/testing/set_data/form_data", {
        familienstand: "married",
        familienstand_married_lived_separated: "no",
        familienstand_confirm_zusammenveranlagung: true,
        person_a_has_disability: "yes",
      });

So yes the next page should be person_a_requests_pauschbetrag.